### PR TITLE
Add ParaAugurDeployFinished event to deployer

### PR DESCRIPTION
**Pull request by justinbarry**

### CODE REVIEW

**Comment by pgebheim** [comment]
We will also need the ParaAugur and ParaAugurTrade addresses as that will power the rest of the augur state syncing.

**Comment by pgebheim** [comment]
@justinbarry -- this needs to be rebased and the contracts re-built

File: packages/augur-core/src/contracts/para/ParaDeployer.sol [hunk 1]
```diff
@@ -199,6 +209,7 @@ contract ParaDeployer is Ownable {
     function deployTrade(address _token) private {
         address _trade = factories.tradeFactory.createTrade();
         paraAugurTradings[_token].registerContract("Trade", _trade);
+        IParaAugur _paraAugur =  paraAugurs[_token];
```
Comment (SwerveFinance) [inline packages/augur-core/src/contracts/para/ParaDeployer.sol:212]: Can be removed I think

**Comment by pgebheim** [comment]
@justinbarry are you combining this PR with #9319 ?

**Pull request closed without merging**